tests/testthat/test-environment.R

```diff
@@ -1,21 +1,24 @@
 test_that("can construct an environment", {
-  env <- new_environment("foo", NULL, NULL)
+  env <- new_environment("foo", NULL, NULL, NULL)
```

Comment:
(note to self) -  the root arg is left unset here, but never gets reffed because sources and globals are both NULL